Hymns/Infrastructure/Data/Models/UiHymn.swift

```diff
@@ -30,6 +30,12 @@ struct UiHymn: Equatable {
         return title + self.title
     }
 
+    var searchableTitle: String {
+        var title = self.title
+        title = title.replacingOccurrences(of: " ", with: "+", options: .literal, range: nil)
+        return title
+    }
+
```

Comment:
I don't think you need this. Just use `toEncodedUrl` to encode it correctly

Reply:
Nice, did this in my last commit. Thanks